### DISCUSSION TITLE
Generate react modules

### DISCUSSION THREAD

**Discussion by markoboras0712** [arminbro/generate-react-cli#89] [Ideas]
Is it possible to create react module with custom module name, which folders are included like "components,hooks,state,.." and files in those folders?

**Comment by arminbro** [comment]
Hey [@markoboras0712](https://github.com/markoboras0712),

Can you show an example of what you're describing?

The combination of [Custom component templates](https://github.com/arminbro/generate-react-cli#custom-component-templates) and [Custom component files](https://github.com/arminbro/generate-react-cli#custom-component-files) should provide the flexibility of generating costume react files.

I hope this is helpful.
